src/univariate/discrete/binomial.jl

```diff
@@ -110,6 +110,18 @@ end
 
 @_delegate_statsfuns Binomial binom n p
 
+function pdf(d::Binomial, x::Real)
+    _insupport = insupport(d, x)
+    s = pdf(d, _insupport ? round(Int, x) : 0)
```

Comment:
is this not recursing without ending (calling pdf again and again)?

Reply:
It shouldn't since `@_delegate_statsfuns` defines `pdf(d::Binomial, ::Int)`. I'll check it again though.

Reply:
It works:
```julia
julia> b = Binomial(10, 0.5)
Binomial{Float64}(n=10, p=0.5)

julia> pdf(b, 0)
0.0009765625

julia> pdf(b, 0.0)
0.0009765625

julia> pdf(b, 1.0)
0.009765625000000002

julia> pdf(b, 1)
0.009765625000000002

julia> pdf(b, 0.5)
0.0
```